Makefile

```
@@ -66,6 +73,14 @@ lint:
 		$(MAKE) vet; \
 	fi
 
+## ── Verify targets (no Docker/Kind/OpenShell needed) ───────────────────
+
+## Fast checks: vet + lint only
+verify-fast: vet lint
```

checkmake (0.3.2)

[warning] Target "verify-fast" should be declared PHONY.

(phonydeclared)